src/spikeinterface/core/generate.py

```diff
@@ -396,8 +396,8 @@ def __init__(
         sampling_frequency: float,
         num_channels: int,
         dtype: Optional[Union[np.dtype, str]] = "float32",
-        seed: Optional[int] = None,
-        mode: Literal['white_noise', 'random_peaks'] = 'white_noise'
+        seed: float = 0,
```

Comment:
I think I prefer to seed None in the signature and set to something when needed only (so almost all tests) to avoid the same siagnls always for a end user.
And I like the idea that user ask themself when a seed the random state.

Reply:
I was testing for a big that I found. I will revert